packages/eclipse/src/styles/globals.css

```
@@ -70,7 +71,9 @@
     --blur-background-low: 16px;
     --blur-background: 24px;
     --blur-background-high: 40px;
-
+    --font-mona-sans:
+        "Mona Sans", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
```

Stylelint (17.3.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)